include/caffe/net.hpp

```diff
@@ -113,6 +116,7 @@ class Net {
   void CopyTrainedLayersFrom(const NetParameter& param);
   void CopyTrainedLayersFrom(const string trained_filename);
   void CopyTrainedLayersFromBinaryProto(const string trained_filename);
+  void CopyTrainedLayersFromBinaryProto(std::istream& trained_stream, const string& name_to_log);
```

Comment:
postfix const.

Reply:
> postfix const.

This is just an extension to caffe code. We have to use caffe coding style.